### ISSUE TITLE
aws-deployment-framework-pipelines fails to generate customdeploy pipeline

### ISSUE THREAD

**Issue by at7925** [awslabs/aws-deployment-framework#142]
aws-deployment-framework-pipelines fails generate customdeploy pipeline, see example:
*custom.yml*
```
---
pipelines:
  - name: cc-customdeploy
    type: cc-customdeploy
    params:
      - SourceAccountId: 295999999999
    targets:
      - path: /deployment
        params:
            Specfile: deployspec-dev.yml
            DeployRoleArn: arn:aws:iam::295999999999:role/adf-cloudformation-deployment-role
```
aws-deployment-framework-pipelines log indicates error YAML not well-formed:
```
Traceback (most recent call last): 
  File "./adf-build/generate_pipelines.py", line 211, in <module> 
    main() 
  File "./adf-build/generate_pipelines.py", line 207, in main 
    thread.join() 
  File "/codebuild/output/src467241165/src/adf-build/python/thread.py", line 30, in join 
    raise self.exc 
  File "/codebuild/output/src467241165/src/adf-build/python/thread.py", line 22, in run 
    **self._kwargs 
  File "./adf-build/generate_pipelines.py", line 154, in worker_thread 
    cloudformation.create_stack() 
  File "/codebuild/output/src467241165/src/adf-build/python/cloudformation.py", line 264, in create_stack 
    create_change_set = self._create_change_set() 
  File "/codebuild/output/src467241165/src/adf-build/python/cloudformation.py", line 186, in _create_change_set 
    self.validate_template() 
  File "/codebuild/output/src467241165/src/adf-build/python/cloudformation.py", line 126, in validate_template 
    raise InvalidTemplateError("{0}: {1}".format(self.template_url, error)) from None 
errors.InvalidTemplateError: https://s3.amazonaws.com/adf-global-base-deployment-pipelinebucket-XXXXXXXX/pipelines/cc-customdeploy/global.yml: An error occurred (ValidationError) when calling the ValidateTemplate operation: Template format error: YAML not well-formed. (line 148, column 31) 
```
checking *global.yml* for error "YAML not well-formed" (line 148, column 31):
```
142   BuildProject:
143     Type: AWS::CodeBuild::Project
144     Properties:
145       Name: !Sub "adf-build-${ProjectName}"
146       Description: !Sub "CodeBuild Project ${ProjectName} created by ADF"
147       EncryptionKey: !ImportValue KMSArn-us-east-1
148       ServiceRole: ServiceRole: !If [HasCustomBuildRole, !Ref CustomBuildRole, !ImportValue CodeBuildRoleArn]
149       Artifacts:
```

**Comment by bundyfx** [comment]
Hi @at7925 

Yep, doesn't look like valid YAML, looks like a copy and paste error on 'ServiceRole:' (duplicated). Would you care to submit a PR to fix?

Thanks!

**Comment by at7925** [comment]
sure, PR will be provided

**Comment by bundyfx** [comment]
Resolved from #142 - Thanks!
